_documentation/user/flags/steamroller.md

```diff
@@ -5,4 +5,26 @@ type: Good
 summary: Destroys tanks you touch but you have to get really close.
 ---
 
-...
+Steamroller flag allows the player to kill tanks that get in a specified proximity of the holder.
+
+## Potency
+
+The effect of killing the opponent automatically if they get close enough can be either really powerful or weak. In the case of Steamroller, the prime factor that decides that is the sole server variable the flag has; *srRadiusMult*.
```

Comment:
This statement needs clarification. The effect of killing the opponent is that they are dead. Perhaps you are referring to the effectiveness of squishing an opponent?

Also, try to describe potency in effects-based terms, not implementation details.